<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWidget</class>
 <widget class="QWidget" name="HomeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="home" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>460</height>
    </rect>
   </property>
   <widget class="QLabel" name="subtitle">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>400</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>系统可能已经存在高危风险</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>200</y>
      <width>200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>立即体检</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbApp">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>390</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbService">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>390</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="WaveBallWidget" name="wallBallWidget" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>190</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>已3天没有体检,建议立即体检!</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbHandler">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>390</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbFix">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>390</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="tbMore">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>390</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="StateWidget" name="fangHuZhongXin" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="StateWidget" name="wangGouXianPei" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>390</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="StateWidget" name="fanLeSuo" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>390</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>460</height>
    </rect>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>home</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveBallWidget</class>
   <extends>QWidget</extends>
   <header location="global">customviews/waveballwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StateWidget</class>
   <extends>QWidget</extends>
   <header location="global">customviews/statewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
